src/main/java/stream/flarebot/flarebot/database/RedisController.java

```diff
@@ -0,0 +1,235 @@
+package stream.flarebot.flarebot.database;
+
+import io.github.binaryoverload.JSONConfig;
+import redis.clients.jedis.Jedis;
+import redis.clients.jedis.JedisMonitor;
+import redis.clients.jedis.JedisPool;
+import redis.clients.jedis.JedisPoolConfig;
+import stream.flarebot.flarebot.FlareBot;
+
+import java.io.IOException;
+import java.util.concurrent.BlockingQueue;
+import java.util.concurrent.LinkedBlockingQueue;
+import java.util.concurrent.TimeUnit;
+
+public class RedisController {
+    private static JedisPool jedisPool;
```

Comment:
New line and make this final

Reply:
Can't make it final as it isn't inited until the constructors hit

Reply:
Exactly, final is init right away or constructor

Reply:
It has to be constructor as theres a config involved, therefore no final

Reply:
What? I see no reason why it can take be final. It's in the constructor and should be a constant, it is practice to make I respect final.

Reply:
*can't

Reply:
Autocorrect is weird...

Reply:
If I make it final, it has to be initialised straight off or there is compilations errors because it's static ![image](https://user-images.githubusercontent.com/15330699/33515747-e10a2746-d75e-11e7-8011-80d75aabe8b3.png)